packages/components/Menu/src/MenuItemRadio/MenuItemRadio.styling.ts

```diff
@@ -0,0 +1,98 @@
+import { Platform } from 'react-native';
+
+import type { Theme, UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+import { borderStyles, fontStyles, layoutStyles } from '@fluentui-react-native/tokens';
+
+import type { MenuItemRadioProps, MenuItemRadioSlotProps, MenuItemRadioTokens } from './MenuItemRadio.types';
+import { menuItemRadioName } from './MenuItemRadio.types';
+import { defaultMenuItemRadioTokens } from './MenuItemRadioTokens';
+
+export const menuItemCheckboxStates: (keyof MenuItemRadioTokens)[] = ['hovered', 'focused', 'pressed', 'disabled', 'checked'];
```

Comment:
Can we change the name here to match the component (menuItemRadioStates)

Reply:
Changed.